design/templates.go

```diff
@@ -0,0 +1,47 @@
+package design
+
+import (
+	_ "goa.design/goa/v3/codegen/generator"
+	. "goa.design/goa/v3/dsl"
+)
+
+var _ = API("templates", func() {
+	Title("Templates Service")
+	Description("A service for applying templates to Keboola projects")
+	Server("templates", func() {
+		Host("localhost", func() {
+			URI("http://localhost:8000")
+		})
+	})
+})
+
+var index = ResultType("application/vnd.templates.index", func() {
```

Comment:
@webrouse Tohle se nelíbí linteru ale není mi úplně jasný jak to vyřešit co nejsprávněji. 🙂
![CleanShot 2022-02-25 at 14 29 05](https://user-images.githubusercontent.com/215660/155723312-182bc77d-fac2-4679-87f0-f0c365c4e459.jpg)

Reply:
Nad `package .. .` -> `// nolint: gochecknoglobals`
Nie je to produkcny kod ale specifikacia, ktora cela funguje na globalnych premennych.

Reply:
ok 👍